charts/datadog/templates/_container-system-probe.yaml

```diff
@@ -55,18 +55,26 @@
       mountPropagation: {{ .Values.datadog.hostVolumeMountPropagation }}
       readOnly: true
     {{- include "linux-container-host-release-volumemounts" . | nindent 4 }}
+  {{- if .Values.datadog.systemProbe.enableDefaultOsReleasePaths }}
+    {{- if ne .Values.datadog.osReleasePath "/etc/redhat-release" }}
     - name: etc-redhat-release
       mountPath: /host/etc/redhat-release
       mountPropagation: {{ .Values.datadog.hostVolumeMountPropagation }}
       readOnly: true
+    {{- end }}
+    {{- if ne .Values.datadog.osReleasePath "/etc/fedora-release" }}
     - name: etc-fedora-release
       mountPath: /host/etc/fedora-release
       mountPropagation: {{ .Values.datadog.hostVolumeMountPropagation }}
       readOnly: true
+    {{- end }}
+    {{- if ne .Values.datadog.osReleasePath "/etc/lsb-release" }}
     - name: etc-lsb-release
       mountPath: /host/etc/lsb-release
       mountPropagation: {{ .Values.datadog.hostVolumeMountPropagation }}
       readOnly: true
+    {{- end }}
```

Comment:
What if the user configures this with `datadog.systemProbe.osReleasePath`? Should we have a helper that looks at both the former option and this one?

Reply:
I knew you will catch it. :)
Yes it should be better if we do it. But I haven't done it because it was existing for a while without complaint. and now `atadog.systemProbe.osReleasePath` is deprecated and undocumented. 
So I was thinking let do not add complexity here for something we want to remove a futur major version